apps/emails-and-messages/src/modules/app-configuration/fallback-app-config.ts

```diff
@@ -0,0 +1,22 @@
+import { AppConfig } from "./app-config";
+import { AppConfigContainer, getDefaultEmptyAppConfiguration } from "./app-config-container";
+import { ChannelFragment, ShopInfoFragment } from "../../../generated/graphql";
+
+/**
+ * TODO Test
+ */
+export const FallbackAppConfig = {
+  createFallbackConfigFromExistingShopAndChannels(
```

Comment:
not this scope but this should happen on app installation, maybe you can create a task for it? We have now hooks to inject that